src/main/java/nl/novi/techiteasyhw/repository/UserRepository.java

```diff
@@ -0,0 +1,8 @@
+package nl.novi.techiteasyhw.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.security.core.userdetails.User;
+
+public interface UserRepository extends JpaRepository<User, String> {
```

Comment:
De User klasse in deze repository komt vanuit de Security.Core.Userdetails.User, Maar je hebt een eigen model Hierdoor werken er bepaalde functies niet in je USER Service (want ook hier importeer je de verkeerde user)